actions/recipe_action.go

```diff
@@ -38,42 +38,45 @@ type RecipeAction struct {
 	Inherits         string
 	Variables        map[string]string
 	Actions          Recipe
-	templateVars     map[string]string
+	context          debos.DebosContext
```

Comment:
I don't think just using another DebosContext makes sense as values in the context can change during Runs.